packages/core/nuxt-theme-module/theme/pages/Category.vue

```diff
@@ -554,4 +556,10 @@ export default {
     width: 100vw;
   }
 }
+.loading {
```

Comment:
it would be more descriptive to put `.loading` in `list` and another `.loading` in `.products` here, wdyt?

Then it's aligned with the CSS hierarchy
![image](https://user-images.githubusercontent.com/15185752/97857983-c6f2bd80-1cfe-11eb-8a8a-39d72c660804.png)